core/src/lib/util.cc

```diff
@@ -584,6 +584,26 @@ const char* job_type_to_str(int type)
   return str;
 }
 
+const char* job_replace_to_str(int replace)
+{
+  const char* str = NULL;
```

Comment:
```suggestion
  // remove the temporary variable and return the from each case level directly
```

Reply:
For consistency, I have applied the style of the existing functions to the new one.